admin/pkg/common/distribution.go

```diff
@@ -0,0 +1,231 @@
+// Copyright 2019 Percona LLC
```

Comment:
shouldn't this file be in the client package?

Reply:
Great catch.
Originally I planned to reuse this file for server as well but I just realized most of it is just client. Moved.